### PR TITLE
Add vector-picture to benchmark tool, and show io/execution gas split

### PR DESCRIPTION
**Pull request by igor-aptos**

### CODE REVIEW

File: aptos-move/block-executor/src/executor.rs [hunk 1]
```diff
@@ -482,25 +482,26 @@ where
                 }
 
                 // Either all txn committed, or a committed txn caused an early halt.
-                if scheduler.halt() {
```
Comment (sitalkedia) [inline aptos-move/block-executor/src/executor.rs:485]: Unrelated change?

File: testsuite/single_node_performance.py [hunk 2]
```diff
@@ -73,51 +73,54 @@ class RunGroupConfig:
 # Local machine numbers will be higher.
 # fmt: off
 TESTS = [
-    RunGroupConfig(expected_tps=25300, key=RunGroupKey("no-op"), included_in=Flow.LAND_BLOCKING),
```
Comment (sitalkedia) [inline testsuite/single_node_performance.py:76]: Unrelated changes?

**Review by sitalkedia — COMMENTED** [review]
Is the plan to add these tests to Forge as well? Especially graffio like txns?

File: aptos-move/block-executor/src/executor.rs (hunk 1 above)
Comment (igor-aptos) [reply aptos-move/block-executor/src/executor.rs:485]: I've decided to revert to previous stage - before it was broken (otherwise I don't get any counters at all), and @ziaptos / @danielxiangzl  can fix it properly

**Comment by igor-aptos** [comment]
yes, we can easily add any individual test to forge - we should see which ones are worth the cost there. 
but maybe we should just have a single with mix of all possible txns we have in the generator in forge, for more comprehensive testing

File: aptos-move/block-executor/src/executor.rs (hunk 1 above)
Comment (igor-aptos) [reply aptos-move/block-executor/src/executor.rs:485]: oh that got landed- let me rebase

**Comment by msmouse** [comment]
So this is not intended to be moderately / very costly txns? (I see we didn't use the SmartTable 😂 )

**Comment by igor-aptos** [comment]
@msmouse next PR , next PR. this is large output txn

**Review by sitalkedia — APPROVED** [review]

File: aptos-move/e2e-testsuite/proptest-regressions/tests/account_universe/peer_to_peer.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Seeds for failure cases proptest has generated in the past. It is
```
Comment (bchocho) [inline aptos-move/e2e-testsuite/proptest-regressions/tests/account_universe/peer_to_peer.txt:1]: Is this meant to be checked in for this PR? Are they legit failures that are happening in main?

File: execution/executor-benchmark/src/lib.rs [hunk 4]
```diff
@@ -495,45 +500,65 @@ fn add_accounts_impl<V>(
...
+            gas,
+            io_gas,
+            execution_gas,
+            gas_count,
+        }
+    }
+}
+
+struct GasMesuring {
+    start: GasMesurement,
```
Comment (bchocho) [inline execution/executor-benchmark/src/lib.rs:543]: Mesur* -> Measur*

File: execution/executor-benchmark/src/lib.rs [hunk 5]
```diff
@@ -599,7 +624,7 @@ mod tests {
 
     #[test]
     fn test_benchmark_transaction() {
-        test_generic_benchmark::<AptosVM>(Some(TransactionTypeArg::TokenV2AmbassadorMint), true);
+        test_generic_benchmark::<AptosVM>(Some(TransactionTypeArg::VectorPicture30k), true);
```
Comment (bchocho) [inline execution/executor-benchmark/src/lib.rs:627]: Are you replacing the previous one? Why not run both?

File: aptos-move/e2e-testsuite/proptest-regressions/tests/account_universe/peer_to_peer.txt (hunk 3 above)
Comment (igor-aptos) [reply aptos-move/e2e-testsuite/proptest-regressions/tests/account_universe/peer_to_peer.txt]: hm. removed from the PR, but let me see if it shows a problem on main, or was from a different branch I had.

File: execution/executor-benchmark/src/lib.rs (hunk 5 above)
Comment (igor-aptos) [reply execution/executor-benchmark/src/lib.rs]: reverted. I use this for testing

might add separately a test for all types or something like that

File: execution/executor-benchmark/src/lib.rs [hunk 6]
```diff
@@ -494,46 +499,66 @@ fn add_accounts_impl<V>(
...
+        Self {
+            gas,
+            io_gas,
+            execution_gas,
+            gas_count,
+        }
+    }
+}
+
+struct GasMesuring {
```
Comment (bchocho) [inline execution/executor-benchmark/src/lib.rs:542]: Can we also fix this one as well? GasMesuring -> GasMeasuring

**Review by bchocho — APPROVED** [review]
